rainforest/tabularvars.go

```diff
@@ -0,0 +1,174 @@
+package rainforest
+
+import (
+	"errors"
+	"strconv"
+	"time"
+)
+
+// Generator is a type representing generators which can be used as variables in RF tests.
+// They can be builtin or uploaded by customer as a tabular variable.
+type Generator struct {
+	ID           int               `json:"id,omitempty"`
+	Name         string            `json:"name,omitempty"`
+	CreationDate time.Time         `json:"created_at, omitempty"`
```

Comment:
We are not using it in our CLI, but it's included in the library, as someone might want to get to that. 😃